### ISSUE TITLE
Can't extend classpath in `start.sh`

### ISSUE THREAD

**Issue by neilstevenson** [hazelcast/hazelcast#13928]
In https://hazelcast.org/download/#imdg
the `start.sh` script has

```
export CLASSPATH="$HAZELCAST_HOME/lib/hazelcast-all-3.10.5.jar"
```

So you can't change the classpath to add JCache.

**Comment by dbrimley** [comment]
Closing this, as start.sh will be replaced with `hazelcast-member` the distribution.  This has all the features requested here.

**Comment by vbekiaris** [comment]
This was actually fixed by #13941 in 3.11; you can now deploy JARs or classes in the `user-lib` directory in Hazelcast distribution and those will be picked up by `start.sh/bat`
